concourse/scripts/environment.sh

```diff
@@ -43,11 +43,13 @@ if [ -z "${CONCOURSE_ATC_PASSWORD:-}" ]; then
   fi
 fi
 
-if [ -n "${DECRYPT_CONCOURSE_ATC_PASSWORD:-}" ]; then
-  GRAFANA_PASSWORD=$(pass "${DECRYPT_CONCOURSE_ATC_PASSWORD}/grafana_admin_password")
-else
-  GRAFANA_PASSWORD=$(aws s3 cp "s3://${DEPLOY_ENV}-state/cf-secrets.yml" - | \
-                     ruby -ryaml -e 'puts YAML.load(STDIN)["secrets"]["grafana_admin_password"]')
+if [ -z "${GRAFANA_PASSWORD:-}" ]; then
+  if [ -n "${DECRYPT_CONCOURSE_ATC_PASSWORD:-}" ]; then
```

Comment:
This variable name is somewhat confusing, it wasn't clear what the concourse ATC password had to do with the Grafana password. I had to look in the makefile to understand what's going on here. I think this could be better named (maybe `PASS_DEPLOY_ENV_DIR` or similar).

I realise that this hasn't been introduced in the PR, but this seems like a good opportunity to address this.